lib/assert.js

```diff
@@ -202,6 +232,13 @@ function objEquiv(a, b, strict) {
     b = pSlice.call(b);
     return _deepEqual(a, b, strict);
   }
+  const aStr = Object.prototype.toString.call(a);
+  const bStr = Object.prototype.toString.call(b);
+  if (aStr === '[object Map]' && bStr === '[object Map]') {
```

Comment:
I guess we cannot measure the equality of WeakMap and WeakSet...
If so, we cannot support WeakMap and WeakSet. would we be better to note the supported types in document?